charts/eks-pod-identity-agent/values.yaml

```diff
@@ -57,7 +57,9 @@ nameOverride: "eks-pod-identity-agent"
 
 nodeSelector: {}
 
-podAnnotations: {}
+podAnnotations:
+  prometheus.io/port: '2705'
+  prometheus.io/scrape: 'true'
```

Comment:
that one I need to fix, annotations is not enough. 

It's a daemonset and taking a node IP, I think setting up hostNetwork:true for this one is not wise from a security perspective

Reply:
it may require maybe a PodMonitor crd, but I'm not sure if the definition for it should be here, since it's a crd and non-default k8s resource

Reply:
I think I could mimic behaviour from a ebs-csi-driver:
https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/charts/aws-ebs-csi-driver/templates/metrics.yaml

Reply:
fixed :) we need to share metric endpoint on 0.0.0.0 so it will be visible for scraping :)